propkatraj/propkatraj.py

```diff
@@ -168,7 +170,7 @@ def __init__(self, atomgroup, select='protein', skip_failure=False,
 
         self.tmpfile = os.path.join(os.path.curdir, 'current.pdb')
         self.skip_failure = skip_failure
-        super(PropkaTraj, self).__init__(self.ag.universe.trajectory, **kwargs)
+        super().__init__(self.ag.universe.trajectory, **kwargs)
```

Comment:
This looks as if this only works in Python 3. Do we properly set the Python version in setup.py and friends?

Reply:
@orbeckst Only mention of python version I can find is an `if` statement in `setup.py`

Reply:
@ianmkenney the setup.py needs updating to include a python_requires (see: https://github.com/MDAnalysis/mdanalysis/blob/c7c5f0a6850d7f63b14b7697134c4cf7dd860fb9/package/setup.py#L640).

The metadata will also need updating as necessary.